spec/generators/install_generator_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "spec_helper"
+
+RSpec.describe Rubyblok::Generators::InstallGenerator do
+  include GeneratorSpec::TestCase
+
+  tests(described_class)
+  destination(File.join(Dir.tmpdir, "files"))
+
+  before(:each) do
+    prepare_destination
+  end
+
+  context "when executing the install generator" do
+    context "when the file doesn't exist" do
+
+      before do
+        run_generator
+      end
+
+      it "should create the initializer file" do
+        assert_file("config/initializers/rubyblok.rb")
```

Comment:
can we test the content of the file? ie
```
assert_file("config/initializers/rubyblok.rb") do |content|
  assert_match(/something/, content)
end
```

probably we need to mock constants like
```
stub_const("STORYBLOK_API_TOKEN", "token")
```